### ISSUE TITLE
Account for calving success in option costs and expected calf sales

### ISSUE THREAD

**Issue by tshrum** [tshrum/drought_decision_model#5]
Currently, all options assume that all cows successfully have calves. Most of the cows who do not successfully calve are likely due to lack of full-term birth or early calf mortality. We should change the options costs to reflect that the herd is probably a bit smaller than twice the number of cows.

Also, do we mean that they sell 75% of living calves in a normal year? Or 75% of 'potential' calves? If they sell 75% of living calves, then we should use that number in calculating expected calf sales. 

Basically, we are over-counting calves in many parts of the model.

**Comment by tshrum** [comment]
Calving success is only used in the calculation of actual sales in Option 3 (sell and replace). This is a problem...

**Comment by tshrum** [comment]
Done. All calf revenue calculations now include a reduction from weaning success. Now we will also be ready to vectorize wn.succ and make it dependent on the rain.
